### PR TITLE
add: User script for Openshift (tested on OCP 4.7.1)

### PR DESCRIPTION
**Pull request by ValentinoUberti**
Issue #229
Added hack/create-user-openshift.sh for creating a demo user and tenant on OCP

### CODE REVIEW

File: hack/create-user-openshift.sh [hunk 1]
```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# This script uses Kubernetes CertificateSigningRequest (CSR) to a generate a
+# certificate signed by the Kubernetes CA itself.
+# It requires cluster admin permission.
+#
+# e.g.: ./create-user.sh alice oil
```
Comment (prometherion) [inline hack/create-user-openshift.sh:7]: Command name should be `create-user-openshift.sh`

**Review by prometherion — CHANGES_REQUESTED** [review]
Just a small remark, once solved LGTM and thanks for testing Capsule on OCP! 🙏

File: hack/create-user-openshift.sh (hunk 1 above)
Comment (ValentinoUberti) [reply hack/create-user-openshift.sh]: Fixed, thank you

**Review by prometherion — APPROVED** [review]
Going to merge once tests are green!

**Pull request merged**